main.css

```diff
@@ -0,0 +1,1029 @@
+    text-align: center;
+    }
+    .badge{
+    border-radius: 1rem;
+    }
+    .badge-lt{
+    border-top-left-radius: 1rem ;
+    border-bottom-left-radius: 1rem ;
+    }
+    .badge-rt{
+    border-top-right-radius: 1rem;
+    border-bottom-right-radius: 1rem;
+    }
+    
+    .badge-round,.badge-sqr{
+    width: 1.5rem;
+    height: 1.5rem;
+    border-radius: 50%;
+    color: var(--primary-white);
+    text-align: center;
+    padding: .1rem;
+    font-size: small;
+    }
+    
+    .badge-sqr{
+    border-radius: 20%;
+    }
+    
+    .badge-online,.badge-offline{
+    background-color: var(--primary-green);
+    border: 2px solid var(--primary-white);
+    border-radius: 50%;
+    width: 1rem;
+    height: 1rem;
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    }
+    
+    .badge-offline{
+    background-color: var(--gray);
+    }
+    
+    /* buttons *********************************************************/
+    
+    .button, a.button, button, [type=submit], [type=reset], [type=button] {
+    display: inline-block;
+    border: 1px solid transparent;
+    padding: 0.75rem 1.25rem;
+    border-radius: 0.25rem;
+    font-family: var(--font-family-sans-serif);
+    font-size: 1rem;
+    font-weight: 600;
+    margin: 0;
+    /* margin-bottom: 0.5rem;
+    margin-right: 0.5rem; */
+    vertical-align: middle;
+    text-align: center;
+    cursor: pointer;
+    text-decoration: none;
+    line-height: 1; 
+    background: var(--primary-blue);
+    color: var(--primary-white);
+    outline: 0;
+    }
+    
+    .button-rnd{
+    border-radius: 2rem !important;
+    }
+    
+    .button-sqr{
+    border-radius: 0rem !important;
+    }
+    
+    /* on hover change the following properties */
+    
+    .button:hover, a.button:hover, button:hover, [type=submit]:hover, [type=reset]:hover, [type=button]:hover {
+    border: 1px solid transparent;  
+    background: var(--secondary-blue);
+    color: var(--primary-white);
+    text-decoration: none; }
+    
+    /* on focus change the following properties */
+    
+    .button:focus, .button:active, a.button:focus, a.button:active, button:focus, button:active, [type=submit]:focus, [type=submit]:active, [type=reset]:focus, [type=reset]:active, [type=button]:focus, [type=button]:active {
+    border: 1px solid transparent;
+    background: var(--secondary-blue);
+    color: var(--primary-white);
+    text-decoration: none; }
+    
+    
+    /* color wale buttons */
+    
+    .button-secondary{
+    background: var(--gray);
+    color: var(--primary-white);
+    }
+    .button-secondary:hover,.button-secondary:focus,.button-secondary:active{
+    background: var(--gray-dark);
+    color: var(--primary-white);
+    }
+    
+    .button-success{
+    background: var(--primary-green);
+    color: var(--primary-white);
+    }
+    .button-success:hover,.button-success:focus,.button-success:active{
+    background: var(--secondary-green);
+    color: var(--primary-white);
```

Comment:
Hi Rakesh, 'color' property is not being changed on hover/success/active. So, this statement should not be required. Please check for other buttons also.